CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 3.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add mockup pattern development support in buildout
```

Comment:
Please descibe what you actually did, not what the title of the issue was.
Think about what yu PR will add and describe it.